frontend/packages/ceph-storage-plugin/integration-tests/tests/add-capacity.scenario.ts

```diff
@@ -0,0 +1,107 @@
+import { execSync } from 'child_process';
+import * as _ from 'lodash';
+import { browser, ExpectedConditions as until } from 'protractor';
+import { appHost } from '@console/internal-integration-tests/protractor.conf';
+import { click } from '@console/shared/src/test-utils/utils';
+import { isNodeReady } from '@console/shared/src/selectors/node';
+import {
+  addCapacityLbl,
+  addCapacityBtn,
+  ocsOp,
+  storageClusterView,
+  storageClusterRow,
+} from '../views/add-capacity.view';
+
+const namespace = 'openshift-storage';
+const kind = 'storagecluster';
+const storageCluster = JSON.parse(
+  execSync(`kubectl get -o json -n ${namespace} ${kind}`).toString(),
+);
+
+let clusterJSON;
+let previousCnt;
+let previousPods;
+let updatedClusterJSON;
+let updatedCnt;
+let updatedPods;
+
+describe('Check add capacity functionality for ocs service', () => {
+  beforeAll(async () => {
+    // check whether the cluster is present or not, and check for its ready state
+    // if not present exit and console that no cluster is present, will be send in next PR
+
+    const { name } = storageCluster.items[0].metadata;
+    clusterJSON = JSON.parse(
+      execSync(`kubectl get -o json -n ${namespace} ${kind} ${name}`).toString(),
+    );
+    previousCnt = _.get(clusterJSON, 'spec.storageDeviceSets[0].count', undefined);
+    const uid = _.get(clusterJSON, 'metadata.uid', undefined).toString();
+    previousPods = JSON.parse(execSync(`kubectl get pods -n ${namespace} -o json`).toString());
+
+    await browser.get(
+      `${appHost}/k8s/ns/openshift-storage/operators.coreos.com~v1alpha1~ClusterServiceVersion`,
+    );
+
+    await click(ocsOp);
+    await click(storageClusterView);
+
+    await browser.wait(until.presenceOf(storageClusterRow(uid)));
+    const kebabMenu = storageClusterRow(uid).$('button[data-test-id="kebab-button"]');
+
+    await click(kebabMenu);
+    await click(addCapacityLbl);
+    await click(addCapacityBtn);
```

Comment:
As per test step, the expansion should be done with IOs in the background. Please make sure that is done.

Reply:
That would be too complex to handle from here.